pom.xml

```diff
@@ -64,15 +64,25 @@
     <changelist>-SNAPSHOT</changelist>
     <skipIntegrationTests>true</skipIntegrationTests>
     <java.level>8</java.level>
-    <jenkins.version>2.60.3</jenkins.version>
-    <jenkins-credentials.version>2.1.11</jenkins-credentials.version>
+    <jenkins.version>2.164.3</jenkins.version>
```

Comment:
I think this will increase the minimal required Jenkins core version. Are there good reasons to do it? 
I'm not sure how many users would find it a problem to upgrade

Reply:
It would be nice if it could work with 2.150 (that's the minimal update center version as per https://jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/)

Reply:
Agreed, I was following the Jenkins-BOM usage guide which have 2.164 as an example https://github.com/jenkinsci/bom#usage

I've modified to 2.150.x